### DISCUSSION TITLE
Disable startup rotation

### DISCUSSION THREAD

**Discussion by diogoc** [roleoroleo/yi-hack-Allwinner-v2#256] [General]
Hello,
It is possible to disable the initial rotation on each axis after the startup?
This way the camera stays allways at the same point when turn on or reboot.

**Comment by roleoroleo** [comment]
The rotation is produced by the yi original software and I can't disable it.
And I think it's necessary to calibrate the position.
Without this rotation, probably presets wouldn't work.

**Comment by diogoc** [comment]
Yes it is for calibration, but for me it's not necessary and it's very annoying.
Unfortunately is not possible...
Thanks

**Comment by ghost** [comment]
Open the camera and disconnect the motor drive cables?

**Comment by diogoc** [comment]
Ok but I still want the camera rotation, only need to disable the startup auto rotation.
